src/diagonal.works/b6/api/functions/collections.go

```diff
@@ -516,3 +518,30 @@ func histogram(co *api.Context, c api.Collection) (api.Collection, error) {
 
 	return &h, nil
 }
+
+func export(_ *api.Context, c api.Collection, filename string) error {
+	file, err := os.OpenFile(filename, os.O_RDWR|os.O_CREATE, 0600)
```

Comment:
there s a TINY BIT of copy / paste between this and yaml helpers in private but ultimately i think it s ok